app/mod_api/__init__.py

```diff
@@ -0,0 +1,25 @@
+from flask import Blueprint
```

Comment:
Can you move this to a *views.py* file and use absolute imports?

Reply:
Yeah, but what do you think about calling it endpoints.py or resources.py?

Reply:
*endpoints.py* is fine

Reply:
I've made the change